robotiq_driver/include/robotiq_driver/hardware_interface.hpp

```diff
@@ -128,7 +128,7 @@ class RobotiqGripperHardwareInterface : public hardware_interface::SystemInterfa
   ROBOTIQ_DRIVER_PUBLIC
   hardware_interface::return_type write(const rclcpp::Time& time, const rclcpp::Duration& period) override;
 
-private:
+protected:
```

Comment:
Just a question. Why do you want to extend the hardware interface?

Reply:
We know of other groups that are looking into merging support for other types of robotiq end effectors. This change will let them test by extending this hardware interface in a minimal way. Maybe we could revisit better abstraction here down the road, but this will unblock folks in the short term.